bootstrap/main.go

```diff
@@ -81,11 +81,20 @@ func mainImpl() {
 
 	// Capture the kubernetes version info to help debug issues
 	fmt.Println("Checking kubectl & kubernetes versions")
-	versionMeta, err := kubectl.GetVersionInfo(kubectlClient)
+	clientVersion, serverVersion, err := kubectl.GetVersionInfo(kubectlClient)
 	if err == nil {
-		raven.SetTagsContext(versionMeta)
+		raven.SetTagsContext(map[string]string{
+			"kubectl_clientVersion_gitVersion": clientVersion,
+		})
+		if serverVersion == "" {
+			exitNoCapture("Error: Could check the kubernetes server version, please check that you can connect to it with \"kubectl version\".\n")
```

Comment:
Did you mean "Could *not*"? Also "check ... check..." reads awkwardly.

Reply:
Good catch!